web_development_101/project_html_css.md

```diff
@@ -1198,310 +1204,312 @@ Submit a link below to the github repo with your files in it by using a pull req
+* [ChiefDakota's solution (Easy)](https://github.com/ChiefDakota/fake-google-it) - [View in Browser](https://chiefdakota.github.io/fake-google-it/)
+* [Kevin's solution (Easy)](https://github.com/super-kebin/google-homepage) - [View in Browser](https://super-kebin.github.io/google-homepage/)
+* [Matt's solution (Easy)](https://github.com/hagnerd/google-homepage/) - [View in Browser](https://hagnerd.github.io/google-homepage/)
+* [Chris' solution (Easy)](https://github.com/medic9r1/google_homepage2) - [View in Browser](https://medic9r1.github.io/google_homepage2/)
+* [bradabayor's solution (Easy)](https://github.com/bradabayor/google-homepage) - [View in Browser](https://bradabayor.github.io/google-homepage)
+* [Elyse Filigheri's solution (Easy)](https://github.com/cuttlefishes/google-homepage) - [View in Browser](https://cuttlefishes.github.io/google-homepage/)
+* [Ivan Martinez's solution(Difficult)](https://github.com/ivanmaru86/google-homepage-Difficult) - [View in Browser](https://ivanmaru86.github.io/google-homepage-Difficult/)
+* [Phuong Hoang's solution (Easy)](https://github.com/phuong01/google-homepage) - [View in Browser](hhttps://phuong01.github.io/google-homepage/)
+* [Bridget Nyirongo's solution (Easy)](https://github.com/Bridget12/google-homepage.git) - [View in Browser](https://bridget12.github.io/google-homepage/)
+* [Luyi Ren's solution (Easy)](https://github.com/LuyiRen/GoogleHomePage) - [View in Browser](https://luyiren.github.io/GoogleHomePage/)
+* [Nikhil Patel's solution (Easy)](https://github.com/nikhilpatel87/google-homepage) - [View in Browser](https://nikhilpatel87.github.io/google-homepage/)
+* [Jamie's solution (Easy)](https://github.com/Zorafins/google-homepage) - [View in Browser](https://zorafins.github.io/google-homepage/)
+* [James Thomson's solution (Easy)](https://bitbucket.org/jthomsonx/google_homepage)
+* [Blake Bryant's solution (Easy)](https://github.com/Balake/google-homepage) - [View in Browser](https://balake.github.io/google-homepage/)
+* [Zach Coursey's solution (Easy)](https://github.com/zcoursey22/google-homepage) - [View in Browser](https://zcoursey22.github.io/google-homepage/)
+* [Doris Keller's solution (Difficult)](https://github.com/DoKe65/google-homepage) - [View in Browser](https://doke65.github.io/google-homepage/)
+* [IvanaGoSt's solution (Easy)](https://github.com/IvanaGoSt/google-homepage) - [View in Browser](https://rawgit.com/IvanaGoSt/google-homepage/master/index.html)
+* [Joe Weston's solution](https://github.com/joeeeeeeeeeeeee/google-homepage) - [View in Browser](https://joeeeeeeeeeeeee.github.io/google-homepage/)
+* [Luke Bartholomew's solution (Easy)](https://github.com/lukesbart/google-homepage) - [View in Browser](https://lukesbart.github.io/google-homepage)
+* [Tony Nyagah's solution (Easy)](https://github.com/TonyNyagah/google_homepage) - [View in Br
```

Comment:
The bullet point formatting on this solution not correct. Please remove the `-` after the `*`